WebDriverAgentLib/Commands/FBDebugCommands.m

```diff
@@ -40,7 +40,7 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleGetSourceCommand:(FBRouteRequest *)request
 {
-  FBApplication *application = request.session.activeApplication;
+  FBApplication *application = request.session.activeApplication? : [FBApplication fb_activeApplication];
```

Comment:
Please change to `request.session.activeApplication ?: [FBApplication fb_activeApplication]`

Reply:
oops, sorry for my mistake... 😅